client/src/components/CategoryShow.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import PastaTile from "./PastaTile.js";
+
+const CategoryShow = (props) => {
```

Comment:
The name of this "feature" or branch is called "create index page", so we really shouldn't be creating show pages here as well